xml/cl.xml

```diff
@@ -2310,6 +2312,12 @@ server's OpenCL/api-docs repository.
             <unused start="0x42A0" end="0x4FFF"/>
     </enums>
 
+    <enums start="0x5000" end="0x500F" name="enums.5000" comment="For cl_ext_buffer_device_address">
+        <enum value="0x5000"        name="CL_MEM_DEVICE_PRIVATE_ADDRESS_EXT"/>
+        <enum value="0x5001"        name="CL_MEM_DEVICE_ADDRESS_EXT"/>
+        <enum value="0x5002"        name="CL_KERNEL_EXEC_INFO_DEVICE_PTRS_EXT"/>
+    </enums>
```

Comment:
I'm fine with this, but a bit of context: AFAICT, the `<enums>` blocks were supposed to be grouping enums by the enum type (like `cl_mem_properties`), but this order was not kept. The `<enums>` block for `cl_device_info` is filled with a ton of random enums from different enum types.
The `comment` attributes on the `<require>` tags below group enums by their type consistently, including in this PR.
And otherwise, I don't see anything of note about this `<enums>` block.

Ah, except you forgot to delete the old definition of `CL_KERNEL_EXEC_INFO_DEVICE_PTRS_EXT`.